supply/lib/supply/uploader.rb

```diff
@@ -25,11 +25,21 @@ def perform_upload
 
       upload_binaries unless Supply.config[:skip_upload_apk]
 
+      promote_track if Supply.config[:track_promote_to]
+
       UI.message("Uploading all changes to Google Play...")
       client.commit_current_edit!
       UI.success("Successfully finished the upload to Google Play")
     end
 
+    def promote_track
```

Comment:
Thanks, great summary! Can you add a small section to the [README](https://github.com/fastlane/fastlane/blob/master/supply/README.md) after **Changelog** with the summaried feature and a short explain of how it can be used?